webserver/include/webserver/thrifthandler.h

```diff
@@ -12,6 +12,7 @@
 #include <util/logutil.h>
 
 #include "mongoose.h"
+#include "webserver/requesthandler.h"
```

Comment:
It is not strictly necessary, but I found it strange that the direct parent of a class defined here is not included.
(Originally Clangd suggested adding it, but since then the rest of the modifications in the file got discarded.)

Should I remove it?

Reply:
In this case, don't remove it, but it should go between angle brackets.